docs/docs/examples/guides/jtc-guide.md

```diff
@@ -200,20 +200,22 @@ The service currently accepts 5 variables, namely:
 
 - `times_from_start`: a list of timestamps (in seconds) measured from the start, indicating when JTC should reach each
 frame.
+- `durations`: a list of absolute durations (in seconds) that correspond to each waypoint (mutually exclusive with 
+`times_from_start`).
 - `frames`: names of the Cartesian frames to reach with the robot end effector 
```

Comment:
frames and joint positions are also technically mutually exclusive. I don't know how readable the expression "mutually exclusive" is, maybe we drop the mutually exclusive stuff and elaborate a bit more precisely in line 214?